static/spec/openapi.yaml

```diff
@@ -86,10 +102,18 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ExportList'
+        '500':
+          description: Failed to get object
```

Comment:
I think we should make the http status code 500 descriptions more generic. Maybe something like "Unexpected server side error".

Reply:
Good idea, yes. Refactored!